cfep-05.md

```diff
@@ -0,0 +1,135 @@
+these labels should generally follow the
+[guidelines](https://docs.python.org/devguide/devcycle.html#stages) that Python
+itself follows.
+
+- "rc": [Beta](https://docs.python.org/devguide/devcycle.html#beta) and [Release
+  Candidate](https://docs.python.org/devguide/devcycle.html#release-candidate-rc)
+  (RC). No new features. Bugfix only.
+
+- "dev": [Pre-Alpha](https://docs.python.org/devguide/devcycle.html#pre-alpha)
+  and [Alpha](https://docs.python.org/devguide/devcycle.html#alpha). These are
+  still packages that could see substantial changes
+  between the dev version and the final release.
+
+If you wish to add numbers to your "dev" or "rc" you should follow the
+[guidelines](http://conda.pydata.org/docs/spec.html#build-version-spec) put
+forth by Continuum regarding version sorting in `conda`. Also see the [source
+code for conda
+4.2.13](https://github.com/conda/conda/blob/4.2.13/conda/version.py#L93-L119).
+The tl;dr here is that conda sorts as follows:
+
+```
+< 1.0
+< 1.1dev1    # special case 'dev'
+< 1.1.0dev1  # special case 'dev'
+== 1.1.dev1   # 0 is inserted before string
+< 1.1.0rc1
+< 1.1.0
+```
+
+So make sure that you ***tag*** your package in such a way that the package name
+that conda-build spits out will sort the package uploaded with an "rc" label
+higher than the package uploaded with the "dev" label.
+
+To create a `dev` or `rc` package a PR can be issued into the `dev` or `rc` branch of the
+feedstock.
+This branch must change the `recipe/conda_build_config.yaml` to point to the `dev_<package_name>` or `rc_<package_name>`.
+
+For example, matplotlib rc releases would include:
+```yaml
+...
+channel_targets:
+  - conda-forge rc_matplotlib
+```
+
+## Alternatives
+
+* Upload pre-release packages to a different anaconda.org user, perhaps
+  `conda-forge-pre`
+* Dump all pre-release packages into one label: "pre"
+
+We understand the proposed approach does not have the same user experience
+as `pip --pre`, which seems simpler and easier to use.
+Currently this level of functionality is not possible with conda while
+maintaining the stability of environment we currently provide.
+This CFEP proposes and initial step toward supporting dev/rc releases.
+As we explore this functinality more we can re-evaluate the user and maintainer
+expreience and how best to provide this functionality.
+
+## FAQ
+
+* What about "alpha" and "beta" labels?
+
+There is not yet heavy use of labels on the conda-forge channel on anaconda.org.
+Given the light usage of labels thus far it seems rather unnecessary to
+introduce many labels.  "dev" and "rc" seem like a nice compromise.
+
+* How do I use labels to install these pre-release packages?
+
+Use the following command, but replace `PACKAGE_NAME` with the package you want
+to install and replace `LABEL` with "rc" or "dev":
+
+```
+conda install -c conda-forge -c conda-forge/label/LABEL_PACKAGE_NAME PACKAGE_NAME
```

Comment:
I would assume that the user will usually want to give the prerelease channel a higher priority, no?
```suggestion
conda install -c conda-forge/label/LABEL_PACKAGE_NAME PACKAGE_NAME -c conda-forge
```